pycoal/cli/environment.py

```diff
@@ -0,0 +1,132 @@
+
+from argparse import ArgumentParser
+from argparse import RawDescriptionHelpFormatter
+
+import logging
+from subprocess import call
+import spectral
+import numpy
+from os.path import abspath, dirname, basename, splitext
+import pycoal
+import time
+sys.path.insert(0, '../pycoal')
+import mineral
+import mining
+import environment
+
+__all__ = []
+
+DEBUG = 1
+TESTRUN = 0
+PROFILE = 0
+
+from pycoal.write import Writer
+from pycoal.read_netcdf import NetCDFReader as Reader
+
+def main(argv=None):
+    '''Command line options.'''
+    logging.basicConfig(filename='pycoal.log', level=logging.INFO, 
+        format='%(asctime)s %(message)s')
+    if argv is None:
+        argv = sys.argv
+    else:
+        sys.argv.extend(argv)
+
+    program_name = os.path.basename(sys.argv[0])
+    program_shortdesc = __import__('__main__').__doc__.split("\n")[1]
+    program_license = '''%s
+  VERSION %s
+  Copyright 2017 COAL Developers. All rights reserved.
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; version 2.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty
+  of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU General Public License for more details.
+  You should have received a copy of the GNU General Public
+  License along with this program; if not, write to the Free
+  Software Foundation, Inc., 51 Franklin Street, Fifth
+  Floor, Boston, MA 02110-1301, USA.
+USAGE
+''' % (program_shortdesc, pycoal.version)
+
+    try:
+        # Setup argument parser
+        parser = ArgumentParser(
+            description=program_license,
+            formatter_class=RawDescriptionHelpFormatter)
+        parser.add_argument(
+            "-m", "--mining",
+            dest="mining_filename",
+            default='ang20150420t182050_corr_v1e_img_class_mining.hdr',
+            help="Input mining classified file to be processed [default: ang20150420t182050_corr_v1e_img_class_mining.hdr]")
+        parser.add_argument(
+            "-hy", "--hydrography",
+            dest="vector_filename",
+            default='Shape/NHDFlowline.shp',
+            help="Path to hydrography data [default: Shape/NHDFlowline.shp]")
+        parser.add_argument(
+            "-e", "--environment",
+            dest="correlation_filename",
+            default='ang20150420t182050_corr_v1e_img_class_mining_NHDFlowline_correlation.hdr',
+            help="Output environmental correlation image [default: ang20150420t182050_corr_v1e_img_class_mining_NHDFlowline_correlation.hdr]")
+
+        # Process arguments
+        args = parser.parse_args(
+            ['-m', 'ang20150420t182050_corr_v1e_img_class_mining.hdr',
+            '-hy', 'Shape/NHDFlowline.shp',
+            '-e', 'ang20150420t182050_corr_v1e_img_class_mining_NHDFlowline_correlation.hdr'])
```

Comment:
Again, there should be no example files, as this is not an example, the user should provide these.